### ISSUE TITLE
0crat can't read claims.xml

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1208]
File `claims.xml` changes may cause `OutOfMemoryError` because it's too big.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@440ba7d5-abf0-4f5c-bcf7-9d1e8f017d75, thread: pool-12-thread-1
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@4bb7975d rejected from java.util.concurrent.ThreadPoolExecutor@6226e10d[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 736]
```

[0.22.29](https://github.com/zerocracy/farm): Issue: zerocracy/farm#1208, Action: `opened`

**Comment by g4s8** [comment]
@0crat release, tag=`0.22.31`

**Comment by g4s8** [comment]
@rultor release, tag=0.22.31

**Comment by g4s8** [comment]
@rultor release, tag=`0.23.1`

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1215](https://github.com/zerocracy/farm/issues/1215) is still not solved.

**Comment by g4s8** [comment]
@g4s8 claims in `PMO` were saved locally and removed. Todo to analyze it was submitted: #1215

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#1245](https://github.com/zerocracy/farm/issues/1245), [#1246](https://github.com/zerocracy/farm/issues/1246) are still not solved; solved: [#1215](https://github.com/zerocracy/farm/issues/1215).

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1246](https://github.com/zerocracy/farm/issues/1246) is still not solved; solved: [#1215](https://github.com/zerocracy/farm/issues/1215), [#1245](https://github.com/zerocracy/farm/issues/1245).

**Comment by 0pdd** [comment]
@g4s8 all 3 puzzles are solved here: [#1215](https://github.com/zerocracy/farm/issues/1215), [#1245](https://github.com/zerocracy/farm/issues/1245), [#1246](https://github.com/zerocracy/farm/issues/1246).
